inshorts.py

```diff
@@ -0,0 +1,95 @@
+# inshorts module contains Inshorts class.
+# inshorts class will be used to fetch the news from inshorts.com as API response for it's instance.
+
+# standard library's
+import requests
+
+
+class Inshorts:
+    """
+    Inshorts module is the main module which will get data from Inshorts API
+    Inshorts class helps in providing 2 varities of News(all_news & Category_news).
+    """
+
+    url = "https://inshorts.com/api/en"
+
+    # def __init__(self,ob_name):
+    #     """Initialize Objects with it's name"""
+    #     self.ob_name = ob_name
+
+    def all_news(self, max_limits=5):
```

Comment:
When mentioning any integers as it is, please use them as constants bcz after the code builts upo with multiple integer values it will be come hard to configure or maintain 
ex: ALL_NEWS_MAX_LIMITS_DEFAULT = 5